modules/@apostrophecms/area/ui/apos/components/AposAreaEditor.vue

```diff
@@ -304,8 +308,8 @@ export default {
         ...this.next.slice(i + 2)
       ];
     },
-    async remove(i) {
-      if (this.docId === window.apos.adminBar.contextId) {
+    async remove(i, { autosave = true } = {}) {
```

Comment:
the widget editor modal now calls back to these methods for preview effects and needs a way to say "don't autosave this, I'm going to clean it up later and revert if the user doesn't go through with it."